docs/scipp-neutron/from-mantid-to-scipp.ipynb

```diff
@@ -784,7 +837,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dspacing = sc.neutron.convert(tof, 'tof', 'd-spacing')"
+    "sc.neutron.convert(da, 'tof', 'd-spacing')"
    ]
   },
   {
```

Comment:
It seems you modified this notebook so that it would be run when the docs are built.
If so, you need to remove the
```
"nbsphinx": {
   "execute": "never"
  }
```
in the notebook metadata, at the end of the notebook.

Reply:
Yes indeed. Thanks for reminding me.

Reply:
It does depend on having `mantid-framework` available too.

Reply:
Yes, but that's fine for the docs